TelligentLocalInstance/Initialize.ps1

```diff
@@ -281,4 +305,134 @@ function Write-Telligent
 	Write-Host
 }
 
+function Expand-Zip {
```

Comment:
Since we're dependant on Powershell 5.0, can we just use `Expand-Archive` instead